yt/funcs.py

```diff
@@ -959,7 +959,7 @@ def enable_plugins(plugin_filename=None):
         # - obsolete config dir.
         my_plugin_name = ytcfg.get("yt", "plugin_filename")
         old_config_dir = os.path.join(os.path.expanduser("~"), ".yt")
```

Comment:
while we're at it, shouldn't this be changed to a getter function too ?

Reply:
getter for what? `~/.yt` was a hardcoded path we used to have.

Reply:
I meant a similar function as config_file() and old_config_dir() that you are introducing in this PR. It's not required, it's just a matter of consistency :)